src/zino/tasks/juniperalarmtask.py

```diff
@@ -58,14 +57,17 @@ async def _get_juniper_alarms(self):
 
         return yellow_alarm_count, red_alarm_count
 
-    def create_alarm_event(self, color: Literal["yellow", "red"], alarm_count: int):
+    def create_alarm_event(self, color: AlarmType, alarm_count: int):
```

Comment:
SOOO MUCH BETTER

Reply:
It was already there, it was just a matter of actually using it.